SmartDeviceLink/SDLLifecycleManager.m

```diff
@@ -367,6 +367,14 @@ - (void)sendManagerRequest:(__kindof SDLRPCRequest *)request withResponseHandler
 - (void)sdl_sendRequest:(SDLRPCRequest *)request withResponseHandler:(nullable SDLResponseHandler)handler {
     // We will allow things to be sent in a "SDLLifeCycleStateTransportConnected" state in the private method, but block it in the public method sendRequest:withCompletionHandler: so that the lifecycle manager can complete its setup without being bothered by developer error
 
+    // If, for some reason, the request is nil we should error out.
+    if (!request) {
```

Comment:
Not sure if we also want in production for it to fail by calling the handler and returning. The assert will only apply during DEBUG. We should probably prevent the error in production?